bin/build_docs.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+sphinx-build -b html "sphinx/source" "docs"
+touch "/docs/.nojekyll"
```

Comment:
I'd rather put this into the makefile. I would not create a bin for doing this, which is rather a rule in the general makefile which call the make rule under the docs folder